client/checkout/utils/copy-test-number.js

```diff
@@ -6,31 +6,37 @@ import { __ } from '@wordpress/i18n';
 document.addEventListener(
 	'click',
 	function ( event ) {
+		if ( ! navigator.clipboard ) {
+			return;
```

Comment:
What do you think about using a fallback mechanism via the `prompt` dialog?

```js
const copyNumberButton = event.target?.closest(
			'.js-woopayments-copy-test-number'
		);
if(!copyNumberButton) {
  return;
}

const number = copyNumberButton.parentElement.querySelector(
				'.js-woopayments-test-number'
			).innerText;

if(!navigator.clipboard) {
  prompt("Copy the test number:", number);
  return;
}
```

Pardon my bad formatting, just writing in GH.

Reply:
Nice idea and I think it works pretty well. Added in dc30ba7288bfd0e1914a11651fb0f96f833f673a. 👍